cookbooks/fb_apache/attributes/default.rb

```diff
@@ -18,6 +33,15 @@
 auth_core_suffix = node.centos6? ? 'default' : 'core'
 
 default['fb_apache'] = {
+  'default_sites_enabled' => true,
+  'default_sites' => [
```

Comment:
I agree. What is the best way to purge unmanaged files from directory? remote_directory doesn't work because of template...